src/Core/src/Platform/iOS/MauiCollectionView.cs

```diff
@@ -0,0 +1,17 @@
+using CoreGraphics;
+using UIKit;
+
+namespace Microsoft.Maui.Platform;
+
+public class MauiCollectionView : UICollectionView
```

Comment:
This is a new API and I am not sure what the rules are for servicing. Maybe this can be internal for now?

Reply:
You are right. There is no reason to have this “public”, I set it to “public” out of habit without thinking.